gnomad_qc/v3/resources/annotations.py

```diff
@@ -1,42 +1,137 @@
-from gnomad.resources.resource_utils import TableResource
+from gnomad.resources.resource_utils import (TableResource,
+                                             VersionedTableResource)
 
-ANNOTATIONS_ROOT = "gs://gnomad/annotations/hail-0.2/ht/genomes_v3"
+from gnomad_qc.v3.resources.constants import CURRENT_RELEASE, RELEASES
 
 
-def get_info(split: bool = True) -> TableResource:
+def _annotations_root(version: str = CURRENT_RELEASE) -> str:
+    """
+    Get root path to the variant annotation files
+
+    :param version: Version of annotation path to return
+    :return: root path of the variant annotation files
+    """
+    return f"gs://gnomad/annotations/hail-0.2/ht/genomes_v{version}"
+
+
+def get_info(split: bool = True) -> VersionedTableResource:
     """
     Gets the gnomAD v3 info TableResource
 
+    :param version: Version of annotation path to return
     :param split: Whether to return the split or multi-allelic version of the resource
-    :return: gnomAD v3 info TableResource
+    :return: gnomAD v3 info VersionedTableResource
     """
-    path = '{}/gnomad_genomes_v3_info{}.ht'.format(
-        ANNOTATIONS_ROOT,
-        '.split' if split else ''
+
+    return VersionedTableResource(
+        CURRENT_RELEASE,
+        {
+            release: TableResource(
+                path="{}/gnomad_genomes_v{}_info{}.ht".format(
+                    _annotations_root(release), release, ".split" if split else ""
+                )
+            )
+            for release in RELEASES
+        },
     )
-    return TableResource(path)
 
 
-def get_filters(model_id: str, split: bool = True) -> TableResource:
+def get_vqsr_filters(
+    model_id: str, split: bool = True, finalized: bool = False,
+) -> VersionedTableResource:
     """
     Gets the specified filtering annotation resource.
 
     :param model_id: Filtering model id
     :param split: Split or multi-allelic version of the filtering file
-    :return: Filtering annotation file
+    :param finalized: Whether to return the raw VQSR table or the finalized VQSR table representing determined cutoffs
+    :return: VQSR filtering annotation file
     """
-    path = '{}/{}_filtering{}.ht'.format(
-        ANNOTATIONS_ROOT,
-        model_id,
-        '.split' if split else ''
+    return VersionedTableResource(
+        CURRENT_RELEASE,
+        {
+            release: TableResource(
+                f"{_annotations_root(release)}/filtering/{model_id}{'.finalized' if finalized else ''}{'.split' if split else ''}.ht"
```

Comment:
Nitpick about consistency of using `format` vs f strings. I find `format` a bit easier to read with the ternary expressions.